### ISSUE TITLE
GFSv16 NCO standards updates

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [NOAA-EMC/UFS_UTILS#138]
According to the latest NCO standards, scripts can not have the ".ecf" extension anymore.  Simply rename it to "exemsfc_global_sfc_prep.sh".

NCO notified us during the code review for GFS v16.  The v16 release branch was updated earlier under #26.

**Comment by GeorgeGayno-NOAA** [comment]
More fallout from the GFS v16 code review - required updates to link_fixdirs.sh:

```
diff --git a/sorc/link_fixdirs.sh b/sorc/link_fixdirs.sh
index 283019d..7f5a37f 100755
--- a/sorc/link_fixdirs.sh
+++ b/sorc/link_fixdirs.sh
@@ -48,7 +48,10 @@ elif [ $machine = "orion" ]; then
 fi
 cd ${pwd}/../fix                ||exit 8
 for dir in fix_am fix_fv3 fix_orog fix_fv3_gmted2010 fix_sfc_climo; do
-    [[ -d $dir ]] && rm -rf $dir
+    if [ -d $dir ]; then
+      [[ $RUN_ENVIR = nco ]] && chmod -R 755 $dir
+      rm -rf $dir
+    fi
     $LINK $FIX_DIR/$dir  .
 done
```

This was added to the v16 release branch.  See #26

**Comment by GeorgeGayno-NOAA** [comment]
Code was frozen and handed to NCO on Oct 9.  These changes are already in the GFS v16 tag.  Now merge them to the authoritative repository.

**Comment by GeorgeGayno-NOAA** [comment]
Merged to 'develop' at 4571564.  Closing.
